microsoft/PowerToys
# Don't use Environment.Exit

## Summary of the Pull Request
- [FZE] don't use Environment.Exit and fix WaitForPowerToysRunner, since it didn't get correct runner PID
- [MeasureTool] Don't use Environment.Exit(0);
- [TextExtractor] Don't use Environment.Exit
- [ColorPicker] Use cancellable NativeEventWaiter + cleanup using
- [PowerLauncher] Remove unused API
- [Awake] Don't use Process.Exit and move to CsWin32, since previously I've had issues with PopupMenu


## PR Checklist

- [ ] **Closes:** Watson crashes


## Detailed Description of the Pull Request / Additional comments


## Validation Steps Performed

- run the module 
- exit PowerToys normally
- observe it exits as well
- do the same but `taskkill /f /im PowerToys.exe`
